File: Python/3-Basic-Actions/8-Click.py
from selenium import webdriver
from selenium.webdriver.common.keys import Keys
from selenium.webdriver.common.by import By

driver = webdriver.Chrome()
driver.implicitly_wait(30)

url = "https://www.starandread.com"

try:
    driver.get(url)
    '''
    Go go starandread.com and Click on "Cart" link on Navbar
    '''
    
    cart_element = driver.find_element(By.XPATH, '//*[@id="bs-example-navbar-collapse-1"]/ul/li[3]/a')
    cart_element.click()

    # Get the title of the cart
    title_element = driver.find_element(By.XPATH, '/html/body/div[3]/center/h4')
    print(title_element.text)

finally:
    driver.quit()

File: Javascript/3-Basic-Actions/8-Click.js
const { Builder, By, Key } = require('selenium-webdriver');

(async function(){
    let driver = await new Builder().forBrowser('chrome').build();
    try{
        (await driver).manage().setTimeouts({implicit:30000});
        await driver.get('https://www.starandread.com/');
        /**
         * Go go google.com and search for "Selenium Webdriver"
         */
        const cart_element = await driver.findElement(By.xpath('//*[@id="bs-example-navbar-collapse-1"]/ul/li[3]/a')); 
        await cart_element.click();

        const title_element = await driver.findElement(By.xpath('/html/body/div[3]/center/h4'));
        console.info(await title_element.getText())
        
    } finally {
        await driver.quit()
    }
})()